packages/core/src/pages/api/preview.ts

```diff
@@ -2,6 +2,13 @@ import type { NextApiHandler, NextApiRequest } from 'next'
 
 import { clientCMS } from 'src/server/cms'
 
+const REDIRECTS_BY_CONTENT_TYPE = {
+  home: '/',
+  plp: '/office',
+  search: '/s?q=headphone',
+  pdp: '/apple-magic-mouse/p',
```

Comment:
what about adding there values in the `faststore.config.js`? we have some LH values there, but i think can be different configs.